doc/0000-ccl_note/main.tex

```diff
@@ -445,14 +306,16 @@ \subsection{Functions of the physical density}
 \item massless neutrino density parameter $\rho_{\nu, {\rm rel}}(a) = \rho_{\rm crit,0} \Omega_{\nu, {\rm rel},0} / a^{4}$,
 \item massive neutrino density parameter $\Omega_{\nu, {\rm m}}(a) = \rho_{\rm crit,0} \Omega_{\nu, {\rm m}}(a) H^2(a) / H_0^2$,
 \end{itemize}
-where $\Omega_{\nu, {\rm m}}(a)$ is given by equation~\ref{Omnu} and the Hubble parameter by equation~\ref{eq:Ha}. \ccl moreover allows for comoving physical densities $\rho_{X, {\rm comoving}}(a) = \rho_X(a) a^3$.
+where $\Omega_{\nu, {\rm m}}(a)$ is given by equation~\ref{Omnu} and the Hubble
+parameter by equation~\ref{eq:Ha}. \ccl moreover allows for comoving physical
+densities $\rho_{X, {\rm comoving}}(a) = \rho_X(a) a^3$.
 
 
 \subsection{Growth function}
 \label{sec:growth}
 
-The routines described in this subsection are implemented in {\tt ccl$\_$background.c}.
```

Comment:
I agree with removing this, but I suggest we make it clear in CONTRIBUTING.md what the structure of the code is, where to add new functions, wrap them, etc. This might already be there. If so, ignore this comment.

Reply:
Most of this information is in CONTRIBUTING.md, but not all of it. I am making a push in another PR to move everything to read the docs. I think it makes sense to punt on this until then, when I plan to write an actual developers guide.